examples/lib/main.dart

```diff
@@ -36,6 +37,7 @@ void main() {
   addComponentsStories(dashbook);
   addEffectsStories(dashbook);
   addExperimentalStories(dashbook);
+  addFlameIsolateExample(dashbook);
```

Comment:
Since this is a bridge package it should be together with the bridge packages in the bottom (line 51)